common/transport/mqtt/test/_mqtt_base_test.js

```diff
@@ -526,5 +526,33 @@ describe('MqttBase', function () {
       });
       fakeMqtt.emit('connect');
     });
+
+    it('disconnects and emits an error if an error is received while connected', function (testCallback) {
+      var fakeMqtt = new FakeMqtt();
```

Comment:
>FakeMqtt [](start = 25, length = 8)

The fakeMqtt class references an instance variable subscribeShouldFail that is only referenced, never set.

Reply:
unrelated but removed.

---
In reply to: [206242605](https://github.com/Azure/azure-iot-sdk-node/pull/343#discussion_r206242605) [](ancestors = 206242605)